### ISSUE TITLE
Early stopping

### ISSUE THREAD

**Issue by AleksCipri** [AleksCipri/DeepMergeDomainAdaptation#6]
Change early stopping to monitor validation loss, wince this is best for preventing overfitting.

**Comment by dkafkes** [comment]
updated to monitor validation loss, but need to figure out why it's stopping before plateau and focus now on fine-tuning learning rate parameters
